miden-store/src/config.rs

```diff
@@ -0,0 +1,43 @@
+use std::fmt::Display;
+
+use serde::{Deserialize, Serialize};
+
+pub const HOST: &str = "localhost";
+// defined as: sum(ord(c)**p for (p, c) in enumerate('miden-store', 1)) % 2**16
+pub const PORT: u16 = 28943;
+pub const ENV_PREFIX: &str = "MIDEN_STORE";
+pub const CONFIG_FILENAME: &str = "miden-store.toml";
+
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
+pub struct Endpoint {
+    /// The host device the server will bind to.
+    pub host: String,
+    /// The port number to bind the server to.
+    pub port: u16,
+}
+
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize, Default)]
+pub struct StoreConfig {
+    /// Main endpoint of the server.
+    pub endpoint: Endpoint,
+}
```

Comment:
Should we also add a path to SQLite db file here?

Reply:
yes, when there will be code to use the db, here is a good place to put it 👍